openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -421,7 +421,7 @@ const Ingestion: React.FC<IngestionProps> = ({
                 className="tw-mr-2"
                 data-testid="airflow-tree-view"
                 disabled={!(permissions.ViewAll || permissions.ViewBasic)}
-                href={`${airflowEndpoint}/tree?dag_id=${text}`}
+                href={`${airflowEndpoint}/tree?dag_id=${record.name}`}
```

Comment:
It might create confusion because in airflow it will be displayed with name and in OM it will be shown with displayName, and if both are the different users will be confused, we must keep the same name at both places!
cc @chirag-madlani what do you think?

Reply:
@ShaileshParmar11 yes, disable the `name` field while editing.